packages/react-component-playground/src/components/welcome-screen.less

```diff
@@ -0,0 +1,20 @@
+.welcome-screen {
+  position: absolute;
+  left: 250px;
+  min-height: 100vh;
+  width: ~"calc(100% - 250px)";
+  display: flex;
+  justify-content: center;
```

Comment:
The `hyphens: auto` is just something I thought would help since it works on all resolutions without awkward right padding.